orte/tools/orterun/orterun.c

```diff
@@ -2414,7 +2414,7 @@ static void setup_debugger_job(void)
      */
     orte_plm_base_create_jobid(debugger);
     /* set the personality to ORTE */
```

Comment:
Should probably update the comment to `ompi` instead of `orte` to reflect the line that follows.